backend/tracim_backend/lib/core/user.py

```diff
@@ -201,15 +201,44 @@ def get_known_users(
         :param exclude_user_ids: user id to exclude from result
         :param exclude_workspace_ids: workspace user to exclude from result
         :param include_workspace_ids: only include users from these workspaces
-        :nb_elem: number of users to return, default value should be low for
-        security and privacy reasons
+        :limit: maximum number of users to return. This value will be capped to
+                KNOWN_MEMBERS_ITEMS_LIMIT if requesting users from workspaces that
+                the requester is not part of.
         :filter_results: If true, do filter result according to user workspace if user is provided
         :return: List of found users
         """
 
-        if len(acp) < 2:
+        nb_elems = KNOWN_MEMBERS_ITEMS_LIMIT
+
+        # RJ - 2020-09-14-NOTE
+        # This is a bit convoluted. Keep in mind that we want to allow:
+        #  - listing a number of members greater than KNOWN_MEMBERS_ITEMS_LIMIT
+        #  - autocomplete less than 2 characters
+        # only if the requesting user is in each and every included workspace.
+        # Otherwise, we don't want to allow that.
+        # By default, we return a maximum of KNOWN_MEMBERS_ITEMS_LIMIT members (when limit is not set),
```

Comment:
Even with your note the behavior of this method is rather unclear.
If you have the use cases of this method in mind, could you write an issue to list them so that we can think of a better answer that only "one method to rule them all"? And link the issue here?